src/content/api/promotion-memberships.mdx

```diff
@@ -99,6 +99,10 @@ Promotion Memberships represents an [Account's](/api/accounts) membership to a [
   <Property name="modifiedBy" type="crn">
     The User or API Key that last modified the Promotion Membership.
   </Property>
+
+  <Property name="test" type="boolean">
+    "true" if the promotion membership is a test resource.
```

Comment:
for consistency, suggest: `true` if the promotion membership is for testing purposes only (copied from loyalty programs)

Reply:
Yep, thanks 🙏